### ISSUE TITLE
The method 'findRenderObject' was called on null. Receiver: null Tried calling: findRenderObject().

### ISSUE THREAD

**Issue by stanojevicboban** [PegasisForever/keep_keyboard_popup_menu#3]
Getting the following fatal exception:

```
Non-fatal Exception: FlutterError
0  ???                            0x0 _getChildRect + 146 (with_keep_keyboard_popup_menu.dart:146)
1  ???                            0x0 <fn> + 202 (with_keep_keyboard_popup_menu.dart:202)
2  ???                            0x0 build + 208 (overlay.dart:208)
3  ???                            0x0 build + 4792 (framework.dart:4792)
4  ???                            0x0 performRebuild + 4675 (framework.dart:4675)
5  ???                            0x0 performRebuild + 4847 (framework.dart:4847)
6  ???                            0x0 rebuild + 4369 (framework.dart:4369)
7  ???                            0x0 update + 4879 (framework.dart:4879)
8  ???                            0x0 updateChild + 3367 (framework.dart:3367)
9  ???                            0x0 updateChildren + 5773 (framework.dart:5773)
10 ???                            0x0 update + 6293 (framework.dart:6293)
11 ???                            0x0 updateChild + 3367 (framework.dart:3367)
12 ???                            0x0 performRebuild + 4700 (framework.dart:4700)
13 ???                            0x0 performRebuild + 4847 (framework.dart:4847)
14 ???                            0x0 rebuild + 4369 (framework.dart:4369)
15 ???                            0x0 buildScope + 2777 (framework.dart:2777)
16 ???                            0x0 drawFrame + 894 (binding.dart:894)
17 ???                            0x0 _handlePersistentFrameCallback + 320 (binding.dart:320)
18 ???                            0x0 _invokeFrameCallback + 1117 (binding.dart:1117)
19 ???                            0x0 handleDrawFrame + 1055 (binding.dart:1055)
20 ???                            0x0 _handleDrawFrame + 971 (binding.dart:971)
```

<img width="559" alt="Screenshot 2021-03-22 at 16 11 24" src="https://user-images.githubusercontent.com/3943008/112012200-49064600-8b29-11eb-82db-898d1c59e8cb.png">

**Comment by PegasisForever** [comment]
Can you reproduce it?

**Comment by stanojevicboban** [comment]
No, this was raised in production mode by one of our users (I can't tell which one, unfortunately). 

The only thing I can think of as to why this is happening is if perhaps the user closed the page while the popup menu was rendering.

**Comment by PegasisForever** [comment]
I added a check to make sure it's not null, this should be fixed now. The new version is published as 1.0.4(non-nullsafety) and 2.0.4(nullsafety)

**Comment by stanojevicboban** [comment]
Thank you @PegasisForever for the quick reaction. I am no longer seeing the issue.
